client/account-recovery/reset-password/reset-password-confirm-form/style.scss

```diff
@@ -0,0 +1,50 @@
+.reset-password-confirm-form__title {
+	font-size: 20px;
+	font-weight: 600;
+	margin-bottom: 16px;
+}
+
+.form-button.reset-password-confirm-form__button {
+	width: 100%;
+	margin: 8px 0;
+}
+
+.reset-password-confirm-form__description {
+	margin: 4px 0;
+}
+
+.form-label.reset-password-confirm-form__text-input-label {
+	display: inline;
+	float: left;
```

Comment:
There shouldn't be any need for `floating` simply stacked blocks. This seems addressing some kind of padding issue, so I'd try to fix it directly, not floating items that would remove them from the flow.

One thing you might want to look into is using `display: flex;` and `flex-direction: column` on the container, so each element is then neatly stacked vertically and you have greater flexibility and control, even if you probably have to change the logic a bit. This isn't a "must" change, however might be valuable for you to look into. Flex is powerful. ;)